rtrlib/rtr/packets.c

```diff
@@ -731,8 +736,9 @@ static int rtr_store_prefix_pdu(struct rtr_socket *rtr_socket, const void *pdu,
 static int rtr_store_router_key_pdu(struct rtr_socket *rtr_socket, const void *pdu, const unsigned int pdu_size, void **ary,
                                     unsigned int *ind, unsigned int *size)
 {
-    const enum pdu_type pdu_type = rtr_get_pdu_type(pdu);
-    assert(pdu_type == ROUTER_KEY);
+    const enum pdu_type type = rtr_get_pdu_type(pdu);
+    (void) type;
+    assert(type == ROUTER_KEY);
```

Comment:
I thought about this shorter version, too. But I did want to keep the 'style' consistent with all other functions and just get rid of the warning.